### PR TITLE
moved files and added sample recipe

### PR DESCRIPTION
**Pull request by SerinityFyre**
Minor changes based on slightly better knowledge of Chef

### CODE REVIEW

**Comment by SerinityFyre** [comment]
complete

File: recipes/sample_recipe.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package 'httpd'
+package 'tree'
+package 'nano'
+
+service 'httpd' do
+        action [:enable, :start]
+end
+
+template '/var/www/html/index.html' do
```
Comment (mhedgpeth) [inline recipes/sample_recipe.rb:9]: are you still using nginx?

**Comment by SerinityFyre** [comment]
Yes, will be using ngnix, just copied over my test recipe so that I could
reference it in a context I would remember

On Oct 3, 2017 8:42 PM, "mhedgpeth" <notifications@github.com> wrote:

*@mhedgpeth* commented on this pull request.
------------------------------

In recipes/sample_recipe.rb
<https://github.com/SerinityFyre/budget_website_infrastructure/pull/2#discussion_r142565394>
:

> @@ -0,0 +1,11 @@
+package 'httpd'
+package 'tree'
+package 'nano'
+
+service 'httpd' do
+        action [:enable, :start]
+end
+
+template '/var/www/html/index.html' do

are you still using nginx?

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/SerinityFyre/budget_website_infrastructure/pull/2#pullrequestreview-66943992>,
or mute the thread

**Comment by mhedgpeth** [comment]
Just saw your email after commenting. This is a great step in the right direction. If you want to take your learning in the Chef direction for a few weeks, that makes sense. You might consider going for the first Chef certification test, which covers all the basics. It would give you a good goal to go after.

**Pull request merged**